src/custom-element.js

```diff
@@ -1597,9 +1597,10 @@ function createBaseCustomElementClass(win) {
         const doc = this.ownerDocument;
         devAssert(doc);
 
-        const container = htmlFor(doc)`
-            <div class="i-amphtml-loading-container i-amphtml-fill-content
-              amp-hidden"></div>`;
+        const html = htmlFor(doc);
+        const container =
+          html`<div class="i-amphtml-loading-container i-amphtml-fill-content
+            amp-hidden"></div>`;
```

Comment:
another idea for saving bytes: this newline and space before it are included in the final build. I'm pretty sure we don't minify extra newlines+whitespace in our tagged templates : (

Reply:
Not at the moment, but we already minify (and remove the extra `raw` literal) but only for single pass. I intentionally only removed leading whitespace to leave the templates readable while we wait for minification. Just a stopgap.